src/main/java/com/radixdlt/crypto/ECKeyUtils.java

```diff
@@ -123,7 +118,7 @@ static void validatePrivate(byte[] privateKey) throws CryptoException {
 		int pklen = privateKey.length;
 		if (allZero(privateKey, 0, pklen - 1)) {
 			byte lastByte = privateKey[pklen - 1];
-			if (lastByte == 0 || lastByte == 1) {
```

Comment:
Is `lastByte == 1` a valid value then?

Reply:
yes, that is the case of PrivateKey of `1`, which is a perfectly valid PrivateKey and used by many test vectors, hence the need for change (and improvement). I use some common secp256k1 test vectors for deterministic signatures using the private key of `1` :).

Reply:
All scalars on interval `1..<N`, where `N` is the order of any elliptic curve is a perfectly valid private key.